generate/schemagen/gen/generator.go

```diff
@@ -70,6 +70,10 @@ func WithFacadeGroups(facadeGroups []FacadeGroup) Option {
 	}
 }
 
+var (
+	structType = reflect.TypeOf(struct{}{})
```

Comment:
should this be called "structType"? It sounds confusing relative to reflect.Struct (which is technically a Kind and not a Type, but I certainly haven't fully internalized that distinction).
This is really about the 'emptyStruct' maybe emptyStructType?